torch/nn/functional.py

```diff
@@ -2316,6 +2316,63 @@ def poisson_nll_loss(input, target, log_input=True, full=False, size_average=Non
     return ret
 
 
+def gaussian_nll_loss(input, target, var, *, eps=1e-6, full=False, reduction='mean'):
+    r"""Gaussian negative log likelihood loss.
+
+    See :class:`~torch.nn.GaussianNLLLoss` for details.
+
+    Args:
+        input: expectation of the Gaussian distribution.
+        target: sample from the Gaussian distribution.
+        var: tensor of positive variance(s), one for each of the expectations
+        in the input (heteroscedastic), or a single one (homoscedastic).
```

Comment:
This is not in your screenshot, but isn't this line missing an extra indent? Other docstring in this file seem to have them.

Reply:
Yup, thanks!